scheduler/scheduler.cpp

```diff
@@ -105,6 +105,7 @@ static list<CompileServer *> css;
 static list<CompileServer *> monitors;
 static list<CompileServer *> controls;
 static list<string> block_css;
+static list<CompileServer *> css_temp_hold;
```

Comment:
I didn't want to use the name 'blacklist' because each CompileServer already has a 'm_blacklist' map. Although this new list applies to all CompileServers, I think it may be possible to just have duplicate data and use those existing blacklists which would be less efficient but potentially less confusing.